cmd/pbapi/main.go

```diff
@@ -76,7 +77,24 @@ func main() {
 	r.Use(m.RequestNum)
 	r.Use(m.MetricsMiddleware)
 	r.Use(m.LoggerMiddleware(&log.Logger))
+
+	// Set Content-Type to JSON for chi and non-chi renderers
+	r.Use(m.ContentTypeMiddleware)
 	r.Use(render.SetContentType(render.ContentTypeJSON))
```

Comment:
Is this second content type setter also useful here now?

Reply:
It actually is, the `SetContentType` only works for Chi renderers, while the other is for non-Chi output. We have few of those, for example OpenAPI JSON and docs.

Reply:
oh, I thought the other will set it for all, not just non-chi, or won't it?

Reply:
From what I've learned, we need both. Headers can be set multiple times, last wins. Now, when you use Chi the render framework has its content type that is stored in context and later set. It overwrites what you set from the normal middleware. The middleware on 83 actually does not set any content type, it stores content type in the context for later use by Chi.

Now, if you feel like setting content type globally to JSON is not what we want, I can move this only to routes which do not use chi, that would be OpenAPI at the moment. I think this might work, unless Chi sets the content type header too late.

Reply:
Yeah that is the case, Chi sets content type the moment you start rendering, it overwrites any value that was there previously: https://github.com/go-chi/render/blob/master/responder.go#L51-L59

The actual set is done here: https://github.com/go-chi/render/blob/master/responder.go#L91-L107

I we have two options:

* Set a global content type just like in my proposal so other non-Chi routes like `/ready` or `/openapi` works fine.
* Delete it from here and move the middleware into all handlers.